R/infrastructure.R

```diff
@@ -121,6 +142,11 @@ use_rcpp <- function(pkg = ".") {
   ignore_path <- file.path(pkg$path, "src", ".gitignore")
   union_write(ignore_path, c("*.o", "*.so", "*.dll"))
 
+  if (use_inst) {
+    use_directory("inst/include/", pkg = pkg)
+    use_template("inst_include_template", "/src/Makevars")
```

Comment:
Ah, and now that I'm looking at it -- should we also generate `src/Makevars.win`?